modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPUtils.macro

```diff
@@ -0,0 +1,45 @@
+definition {
+
+	macro addEVPSite {
+		Site.openSitesAdmin();
+
+		TeamExtranet.addSiteTemplateCP(
+			siteName = "EVP",
+			siteTemplateName = "EVP");
+	}
+
+	macro assertPageTitle {
+		AssertElementPresent(
+			key_tabTitle = ${tabTitle},
+			locator1 = "EVPUtils#TAB_TITLE");
+	}
+
+	// @summary = "Go to Folder"
```

Comment:
Please remove comments if the macro is needed.
If the macro is not yet needed, please delete.